packages/augur-core/.gitignore

```diff
@@ -14,6 +14,7 @@ __pycache__/
 .env/
 env/
 venv/
+venv/
```

Comment:
`Hex f00d` what is this?

Reply:
This is how the folder is autogenerated in Windows